flutter_app/lib/Utils/ApplicationUtils.dart

```diff
@@ -200,12 +216,31 @@ class ApplicationUtils implements ExotelSDKCallback {
   }
 
   @override
-  void onCallIncoming(Map<String, String> arguments) {
-    String? callId = arguments['callId'];
-    String? destination = arguments['destination'];
-    setCallId(callId!);
-    setDestination(destination!);
+  void onCallIncoming(String callId, String destination) {
+    print("in onCallIncoming application utils");
+    setCallId(callId);
+    setDestination(destination);
     navigateToIncoming();
+    _fetchData(mDestination!, 'INCOMING');
```

Comment:
what this funtion actually do?
and rename the funtion which is more suitable to its functionality.